chains/orchestrator-relays/runtime/dancelight/src/lib.rs

```diff
@@ -1823,6 +1840,12 @@ construct_runtime! {
         ExternalValidatorSlashes: pallet_external_validator_slashes = 21,
         ExternalValidatorsRewards: pallet_external_validators_rewards = 22,
 
+        // Bridging stuff - 1
+        EthereumOutboundQueue: snowbridge_pallet_outbound_queue = 23,
+        EthereumInboundQueue: snowbridge_pallet_inbound_queue = 24,
+        EthereumSystem: snowbridge_pallet_system = 25,
+        OutboundMessageCommitmentRecorder: pallet_outbound_message_commitment_recorder = 26,
```

Comment:
Can you add an integration test for this? Something that will break if someone changes the pallet order back to what it was before

Reply:
Okay. will do.

Reply:
So, I tried to write integration test for this, but it turns out that will be covered as part of the rewards integration test.

Reply:
*disregard*